src/Rokt-Kit.js

```diff
@@ -74,7 +94,46 @@ var constructor = function () {
         }
     }
 
+    function selectPlacements(options) {
+        var attributes = (options && options.attributes) || {};
+        var placementAttributes = mergeObjects(attributes, self.userAttributes);
```

Comment:
i would flip these, so that the attributes passed in from the customer override anything we have cached.